app/lib/features/chat/widgets/convo_list.dart

```diff
@@ -43,18 +46,31 @@ class _ConvosListConsumerState extends ConsumerState<ConvosList> {
           );
     }
 
-    if (chats.isEmpty) {
+     if (chats.isEmpty) {
       return Center(
-        heightFactor: 10,
-        child: Text(
-          '${AppLocalizations.of(context)!.loadingConvo}...',
+        heightFactor: 1.5,
+        child: ErrorWidgetTemplate(
+          title: 'You have no DMs at the moment',
+          subtitle:
+              'Get in touch with other change makers, organizers or activists and chat directly with them.',
+          image: 'assets/images/empty_chat.png',
+          primaryButton: DefaultButton(
+            onPressed: () {},
```

Comment:
Pretty sure we agreed to not add buttons without function anymore. Please either remove the button for now or better route to the actual activity.

Reply:
Fixed the routing issues for the non functional buttons .